### DISCUSSION TITLE
On-Demand, NOT BAD

### DISCUSSION THREAD

**Discussion by 2q2code** [moghtech/komodo#181] [Q&A]
I currently run a few servers. Most of them are always-on, but two are on-demand, and they power down when not in use. They are triggered on by WOL via the load balancer, which runs Caddy. I would like to be sure I have set them up correctly in Komodo.[^1]

At present, I can turn off alerts, but the server still comes up _red_ in the dashboard and there is still a notification dot which distracts my brain. So I have disabled the on-demand servers for the moment. They come up a pleasing blue instead, but I suspect they will need to be enabled again when they are online to use with the web-admin tool.

Ideally, I'd like a way of marking a server as _on-demand_ so if it goes offline, Komodo knows that it's allowed to do that, and not to freak out. It could show the server and resources as _offline_ or something.

[^1]: I have just migrated (today!) from Portainer and its agents. So far, I am extremely happy with the smooth transition. All the stacks migrated well (one at a time) and the web-ui was intuitive and helpful. A few minor things could have made things easier, but overall, I am vary satisfied with my decision to move to Komodo.

**Comment by mbecker20** [comment]
Good point. I think the only issue here is the notification needs to be able to be disabled in UI (you can already turn off "sending" the alert message to eg discord if you configure an Alerter)

**Comment by mbecker20** [comment]
There kind of needs to be a new server status, if the server is enabled but unreachable it will show big red NOT OK. In this case, the server is enabled, unreachable, but it kind of is OK, maybe yellow `DOWN` status would work.

**Reply by 2q2code** [reply]
I would suggest that yellow is still a warning colour. As an ex designer (among other things), I wouldn't want to tie it to alarm. UP and DOWN are fine terminologies to use because they are accurate, but DOWN does have negative connotations that require action (the server is down, the system is down, the internet is down, your fly is down), so perhaps using OFFLINE is more language and intent neutral. If a server is not explicitly expected to be in the OFFLINE state, the colour could be red, but if a server has the ON_DEMAND flag set, the colour could be something more relaxed - like a lavender or soft violet.
